tex/complex-ana/quintic.tex

```diff
@@ -0,0 +1,187 @@
+\chapter{Bonus: Topological Abel-Ruffini Theorem}
+
+We've already shown the Fundamental Theorem of Algebra.
+Now, with our earlier intuition on holomorphic $n$th roots,
+we can now show that there is no general formula for 
+the roots of a quintic polynomial.
+
+\section{The Game Plan}
+
+Firstly, what do we even mean by ``formula'' here?
+
+\begin{definition}
+    A \vocab{quintic formula} would be a formula taking in the coefficients $(a, \dots, f)$ of a 
+    degree $5$ polynomial $P$, using the operations $+, -, \times, \div, \sqrt[n]{}$ 
```

Comment:
the typesetted output doesn't look quite right. Maybe try adding a `\cdot` inside?

Reply:
I've done that.